vendor/modules.txt

```diff
@@ -91,9 +91,11 @@ github.com/container-storage-interface/spec/lib/go/csi
 # github.com/csi-addons/replication-lib-utils v0.2.0
 ## explicit; go 1.15
 github.com/csi-addons/replication-lib-utils/protosanitizer
-# github.com/csi-addons/spec v0.1.2-0.20211220083702-c779b23cf97c
+# github.com/csi-addons/spec v0.1.2-0.20211220115741-32fa508dadbe
 ## explicit
+github.com/csi-addons/spec/lib/go/fence
 github.com/csi-addons/spec/lib/go/identity
+github.com/csi-addons/spec/lib/go/reclaimspace
```

Comment:
why reclaim is getting added here?

Reply:
As discussed with @nixpanic , in go version 1.17, it seems to be adding those dependencies, which it was not added in the reclaimspace PR as it was implemented in a previous version.
As suggested by @nixpanic, added a new commit to cleanup the vendor.
Thanks!